backend/src/main/java/com/bakdata/conquery/models/query/queryplan/ConceptQueryPlan.java

```diff
@@ -81,9 +81,9 @@ protected void checkRequiredTables(WorkerStorage storage) {
 		}
 	}
 
-	public void init(Entity entity) {
+	public void init(Entity entity, QueryExecutionContext ctx) {
 		this.entity = entity;
```

Comment:
warum gibt es hier den member `entity`, der letztendlich in `execute` nicht benutzt wird.

Reply:
Das ist diese misch-masch OOP Implementierung aller QueryEngine Objekte, aber guter Punkt, muss mal schauen, wie man die smarter setzen kann, wird ja schon benutzt

Reply:
Habe gerade versucht die init vor den execute schritt zu schieben, aber da gibt es echt viele abhängigkeiten vA in den Formularen, dass das am Ende echt nur uneinheitlicher als jetzt ist